pkg/registry/registry.go

```diff
@@ -2,15 +2,17 @@ package registry
 
 import (
 	"fmt"
+	nurl "net/url"
 	"time"
 
+	dockerRef "github.com/docker/distribution/reference"
 	dockerclient "github.com/docker/docker/client"
-	"github.com/giantswarm/backoff"
-	"github.com/giantswarm/microerror"
-	"github.com/giantswarm/micrologger"
 	"github.com/nokia/docker-registry-client/registry"
 	"github.com/opencontainers/go-digest"
 
+	"github.com/giantswarm/backoff"
```

Comment:
Not sure where this is coming from: We generally have 3 blocks: 
1. Standard lib imports
2. External imports
3. Project internal imports

So these imports belong into the second block:
https://github.com/giantswarm/fmt/blob/master/go/imports.md

Reply:
VS import organization /shrug